domains/client/block-preprocessor/src/stateless_runtime.rs

```diff
@@ -378,4 +409,15 @@ where
             inner_call,
         )
     }
+
+    pub fn construct_evm_contract_creation_allowed_by_extrinsic(
+        &self,
+        inner_call: PermissionedActionAllowedBy<EthereumAccountId>,
+    ) -> Result<Block::Extrinsic, ApiError> {
+        <Self as EvmTrackerApi<Block>>::construct_evm_contract_creation_allowed_by_extrinsic(
+            self,
+            Default::default(),
+            inner_call,
+        )
```

Comment:
We also need to check runtime api version to see if this API exist in the domain runtime code